package.json

```diff
@@ -136,7 +136,8 @@
             "kotlin": "cd $dir && kotlinc $fileName -include-runtime -d $fileNameWithoutExt.jar && java -jar $fileNameWithoutExt.jar",
             "dart": "dart",
             "pascal": "cd $dir && fpc $fileName && $dir$fileNameWithoutExt",
-            "d": "cd $dir && dmd $fileName && $dir$fileNameWithoutExt"
+            "d": "cd $dir && dmd $fileName && $dir$fileNameWithoutExt",
+            "haskell": "runhaskell"
```

Comment:
You also need to add ".hs" definition in "code-runner.executorMapByFileExtension", since the `haskell` is not a default language id provided by VS Code.